src/movies.js

```diff
@@ -1,16 +1,85 @@
 // Iteration 1: All directors? - Get the array of all directors.
+
+function getAllDirectors(allMovies) {
+    const result = allMovies.map(elm => {
+        return elm.director
+    })
+    return result
+}
```

Comment:
Esta y otras se pueden refactorizar así, no es obligatorio pero para que te vayas acostumbrando.
```suggestion
const getAllDirectors = (allMovies) => allMovies.map(elm => elm.director)
```